Dockerfile

```diff
@@ -14,8 +14,7 @@ RUN --mount=type=cache,target=/root/.cache/uv \
 # Copy the project into the intermediate image
 COPY pyproject.toml pyproject.toml
 COPY uv.lock uv.lock
-COPY run_server.py /app/run_server.py
-COPY app.py /app/app.py
+COPY server /app/server
```

Comment:
Following what we do for PMA here
https://github.com/gnosis/prediction-market-agent/blob/main/Dockerfile#L30